spec/factories/clearance.rb

```diff
@@ -4,7 +4,16 @@
   end
 
   factory :user do
+    name "Jon Smith"
     email
     password "password"
+
+    trait :admin do
+      admin true
+    end
```

Comment:
You may want to use a nested factory here instead so you can do `create :admin`. Same for the patron trait